doc/nrf/releases/release-notes-1.5.0.rst

```diff
@@ -0,0 +1,1132 @@
+    See ``CONFIG_BOOT_FIH_PROFILE_HIGH`` and similar Kconfig options.
+  * Introduced abstract crypto primitives to simplify porting.
+  * Added ram-load upgrade mode (not enabled for Zephyr yet).
+  * Renamed single-image mode to single-slot mode.
+    See the ``CONFIG_SINGLE_APPLICATION_SLOT`` option.
+  * Added a patch for turning off cache for Cortex-M7 before chain-loading.
+  * Fixed an issue that caused HW stack protection to catch the chain-loaded application during its early initialization.
+  * Added reset of Cortex SPLIM registers before boot.
+  * Fixed a build issue that occurred if the CONF_FILE contained multiple file paths instead of a single file path.
+  * Added watchdog feed on nRF devices.
+    See the ``CONFIG_BOOT_WATCHDOG_FEED`` option.
+  * Removed the ``flash_area_read_is_empty()`` port implementation function.
+  * Updated the ARM core configuration to only be initialized when selected by the user.
+    See the ``CONFIG_MCUBOOT_CLEANUP_ARM_CORE`` option.
+  * Allowed the final data chunk in the image to be unaligned in the serial-recovery protocol.
+  * Updated the ``CONFIG_BOOT_DIRECT_XIP_REVERT`` option to be valid only in xip-mode.
+  * Added an offset parameter to the tinycrypt ctr mode so that it can be properly used as a streaming cipher.
+  * Configured the bootloader to use a minimal CBPRINTF (:option:`CONFIG_CBPRINTF_NANO`) implementation.
+  * Configured logging to use :option:`CONFIG_LOG_MINIMAL` by default.
+  * Fixed a vulnerability with nokogiri<=1.11.0.rc4.
+  * Introduced a bootutil_public library that contains code common to MCUboot and the DFU application.
+    See :option:`CONFIG_MCUBOOT_BOOTUTIL_LIB`.
+
+* Image tool:
+
+  * Updated the tool to print an image digest during verification.
+  * Added a possibility to set a confirm flag for HEX files as well.
+  * Updated the usage of ``--confirm`` to imply ``--pad``.
+  * Fixed the argument handling of ``custom_tlvs``.
+  * Added support for setting a fixed ROM address in the image header.
+
+Mcumgr
+======
+
+The mcumgr library contains all commits from the upstream mcumgr repository up to and including snapshot ``74e77ad08``.
+
+The following list summarizes the most important changes inherited from upstream mcumgr:
+
+* Fixed an issue with devices running MCUboot v1.6.0 or earlier where a power outage during erase of a corrupted image in slot 1 could result in the device not being able to boot.
+  In this case, it was not possible to update the device and mcumgr would return error code 6 (``MGMT_ERR_EBADSTATE``).
+* Added support for invoking shell commands (shell management) from the mcumgr command line.
+* Added optional verification of an uploaded direct-xip binary, which will reject any binary that cannot boot from the base address of the offered upload slot.
+  This verification can be enabled through :option:`CONFIG_IMG_MGMT_REJECT_DIRECT_XIP_MISMATCHED_SLOT`.
+
+
+Build system
```

Comment:
This should probably be a subsection to Zephyr?
Or even added to the list under line 848?

@carlescufi or @mbolivar-nordic ?

Reply:
> This should probably be a subsection to Zephyr?
> Or even added to the list under line 848?

In my opinion, the NCS build system is a separate topic from the Zephyr build system, so in general I would expect changes to some build system topics to be outside of the Zephyr section.

However, these changes are indeed upstream only, so I agree they could be moved to the Zephyr section.

Reply:
Moved to the Zephyr section.